service/history/api/updateworkflowoptions/api_test.go

```diff
@@ -57,65 +56,75 @@ var (
 
 func TestMergeOptions_VersionOverrideMask(t *testing.T) {
 	updateMask := &fieldmaskpb.FieldMask{Paths: []string{"versioning_override"}}
-	opts := &workflowpb.WorkflowExecutionOptions{}
+	input := &workflowpb.WorkflowExecutionOptions{}
 
 	// Merge unpinned into empty options
-	opts, err := applyWorkflowExecutionOptions(opts, unpinnedOverrideOptions, updateMask)
+	merged, err := applyWorkflowExecutionOptions(input, unpinnedOverrideOptions, updateMask)
 	if err != nil {
 		t.Error(err)
 	}
-	assert.True(t, proto.Equal(unpinnedOverrideOptions, opts))
+	assert.EqualExportedValues(t, unpinnedOverrideOptions, merged)
```

Comment:
Using `EqualExportedValues` will show you the diff; which is handy!

Reply:
nice!